src/drive/web/modules/drive/RenameInputV2.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { withClient } from 'cozy-client'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+
+import FilenameInput from 'drive/web/modules/filelist/FilenameInput'
+import { abortRenaming } from './renameV2'
+
+const updateFileNameQuery = async (client, file, newName) => {
+  return client
+    .collection('io.cozy.files')
+    .updateAttributes(file.id, { name: newName })
```

Comment:
I think it'd be better to use `client.save()` here as the document would be updated in the store without waiting for the realtime.

```
client.save({ ...file, name: newName })
```

Reply:
Yep, we're doing it ;).